frontend/src/views/MyProgress/summary/InformationMeasureSummary.vue

```diff
@@ -1,11 +1,117 @@
 <template>
-  <div>
-    Info summary
+  <div class="fr-text">
+    <ul>
```

Comment:
```suggestion
    <ul role="list">
```
https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#accessibility_concerns